src/index.tsx

```diff
@@ -10,7 +10,7 @@ import { configureStore } from './store';
 initApi({
   host:
     process.env.NODE_ENV === 'production'
-      ? 'http://koku-koku.1b13.insights.openshiftapps.com/'
+      ? 'http://koku-koku-dev.1b13.insights.openshiftapps.com'
```

Comment:
Tried utilizing `process.env.APP_NAMESPACE` instead of hard-coding `koku-dev` here but it didn't seem to pick up the environment variable from from the ConfigMap.

<img width="946" alt="image" src="https://user-images.githubusercontent.com/29379759/43218221-c5004a64-9011-11e8-889e-59c9b6ec7aae.png">


<img width="454" alt="image" src="https://user-images.githubusercontent.com/29379759/43218246-d5859e98-9011-11e8-9191-3f7086a7736e.png">

I figure we could follow up with a secondary PR to get this so it will work in staging too.

Reply:
We have to explicitly add environment variables to the client.  We can update that in another PR.